webvtt.go

```diff
@@ -203,6 +205,21 @@ func ReadFromWebVTT(i io.Reader) (o *Subtitles, err error) {
 	return
 }
 
+func parseTextWebVTT(i string) (o Line) {
+	match := voiceTagRegexp.FindStringSubmatch(i)
+
+	endTag := match[5]
```

Comment:
You need to check that `len(match)>5`

Reply:
with the current regexp , the match length is always 6. so it is not really useful to check the length here.